src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const baseRate = 40;
+  const sevenDaysDiscount = 50;
+  const threeDaysDiscount = 20;
+
+  if (days >= 7) {
+    return baseRate * days - sevenDaysDiscount;
```

Comment:
don't repeat yourself: create separate variable for baseRate * days value

Reply:
ok